crates/rethnet_rpc_server/src/lib.rs

```diff
@@ -75,77 +74,76 @@ where
     }
 }
 
+async fn get_latest_block_number<T>(state: &StateType) -> Result<U256, ResponseData<T>> {
+    if let Some(fork_block_number) = state.fork_block_number {
+        // TODO: when we're able to mint local blocks, and there are some newer than
```

Comment:
Could you please create a GitHub issue for this and link to it?

Reply:
addressed in https://github.com/NomicFoundation/hardhat/pull/3998/commits/15c7cc92d0a9bbd3bcd435f52c12ad97bb684904